modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` field in the revision template.
+
+Here is an example of `target` and `containerConcurrency` being used in a revision template.
+
+[source,yam]
+----
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: myapp
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/target: 50
+    spec:
+      containerConcurrency: 2
+      containers:
+      - image: myimage
+----
+
+[NOTE]
```

Comment:
It looks like it's rendering okay, and maybe this syntax is acceptable, but the normal syntax for notes that we use has four ==== surrounding the content.

```
[NOTE]
====
Text
====
```

Reply:
Thanks Andrea!